modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/PicklistObjectFieldBusinessType.java

```diff
@@ -177,9 +170,23 @@ public void predefineObjectFieldSettings(
 			ObjectStateFlow objectStateFlow =
```

Comment:
@G-Sa rename it to `newObjectStateFlow` to follow the same pattern from `ObjectField newObjectField`